ocp-build-data-validator/validator/json_schemas/shipment.schema.json

```
@@ -259,12 +267,18 @@
       "type": "object"
     },
     "ShipmentEnv": {
+      "additionalProperties": false,
       "description": " Environment specific configuration for a release ",
       "properties": {
         "releasePlan": {
           "title": "Releaseplan",
           "type": "string"
         },
+        "liveID": {
+          "default": null,
+          "title": "Liveid",
```

```suggestion
          "title": "liveID",
```
optionally, for consistency

---

this is auto-generated by the schema dump